README.md

```diff
@@ -119,6 +119,21 @@ When integrating with the ORCA system, the EHR (and its FHIR API) needs to suppo
 - *OPTIONAL*: If using the *Frontend*, invoking *Orchestrator*'s app launch
   - Supported: SMART on FHIR (TODO), ChipSoft HIX (TODO)
 
+### Orca Care Plan Contributor task notification
+When a new Task is created or updated at the Care Plan Service, the Task filler's EHR will be notified. As soon as the 
+is accepted, the Task filler's EHR will be notified. The notification is sent to the EHR's by Kafka / EventHubs. The 
+notification mechanism will change over time:
+1. The initial implementation will use Kafka / EventHubs to notify the EHR's of new Tasks and contain all relevant 
+resources in one sibling message. This message contains of a set of FHIR bundles.
+2. The future implementation will use Kafka / EventHubs to notify the EHR's of new Tasks without the relevant FHIR 
+resources as the EHR's should query the Care Plan Service for the relevant resources.
```

Comment:
A bit 'wollig', and we're sending just 1 message, right? Maybe:

```suggestion
When the local Task filler accepts a new Task, ORCA will notify the EHR so it can process it. The following messaging protocols are supported:
- Kafka
- Azure Event Hub (through Kafka)

Configure the `ORCA_CAREPLANCONTRIBUTOR_KAFKA` properties to enable delivery through Kafka.

The message contains the Task and several FHIR bundles that contain the FHIR resources related to the Task.
Note: the bundles with the related FHIR resources might be removed in future.
```